packages/plugin-infra/src/manager/dashboard.ts

```diff
@@ -27,5 +28,16 @@ export function useActivatedPluginDashboard(pluginID: string) {
 }
 
 export function startPluginDashboard(host: Plugin.__Host.Host<Plugin.Dashboard.DashboardContext>) {
-    startDaemon(host)
+    startDaemon(host, (id) => {
+        const def = getPluginDefine(id)
+        if (!def) return false
+
+        const currentNetwork = getSiteType()
+        if (!currentNetwork) return false
+
+        const status = def.enableRequirement.supports.sites[currentNetwork]
+        if (def.enableRequirement.supports.type === 'opt-in' && status !== true) return false
+        if (def.enableRequirement.supports.type === 'opt-out' && status === true) return false
```

Comment:
you should not do this check in the dashboard. dashboard plugin host only start in one place, which is dashboard, and it does not need this type of check.